### PR TITLE
whats-new-in-storage - USB flash drive as a witness: Specifies direct links to relevant documentation

### PR DESCRIPTION
**Pull request by exchange12rocks**

### CODE REVIEW

**Comment by PRMerger16** [comment]
@exchange12rocks : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: WindowsServerDocs/storage/whats-new-in-storage.md [hunk 1]
```diff
@@ -75,7 +75,7 @@ There are a number of improvements to Storage Spaces Direct in Windows Server 20
 
 - **Two-server clusters using a USB flash drive as a witness**
 
-    Use a low-cost USB flash drive plugged into your router to act as a witness in two-server clusters. If a server goes down and then back up, the USB drive cluster knows which server has the most up-to-date data. For more info, see the [Storage at Microsoft blog](https://techcommunity.microsoft.com/t5/storage-at-microsoft/bg-p/FileCAB).
+    Use a low-cost USB flash drive plugged into your router to act as a witness in two-server clusters. If a server goes down and then back up, the USB drive cluster knows which server has the most up-to-date data. For more info, see the [Storage at Microsoft blog](https://techcommunity.microsoft.com/t5/storage-at-microsoft/here-s-what-you-missed-8211-five-big-announcements-for-storage/ba-p/428257) and [documentation on how to deploy a file share witness](../failover-clustering/file-share-witness#creating-a-file-share-witness-on-a-router-with-a-usb-device.md).
```
Comment (illfated) [inline WindowsServerDocs/storage/whats-new-in-storage.md:78]: Please note: The filename extension `.md` belongs to the filename, not the page anchor (as far as I know):
```suggestion
    Use a low-cost USB flash drive plugged into your router to act as a witness in two-server clusters. If a server goes down and then back up, the USB drive cluster knows which server has the most up-to-date data. For more info, see the [Storage at Microsoft blog](https://techcommunity.microsoft.com/t5/storage-at-microsoft/here-s-what-you-missed-8211-five-big-announcements-for-storage/ba-p/428257) and [documentation on how to deploy a file share witness](../failover-clustering/file-share-witness.md#creating-a-file-share-witness-on-a-router-with-a-usb-device).
```
Tested and found to be working both in this GH repository and on docs.microsoft.com (without the `.md` extension).

**Comment by PRMerger13** [comment]
@exchange12rocks : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: WindowsServerDocs/storage/whats-new-in-storage.md (hunk 1 above)
Comment (exchange12rocks) [reply WindowsServerDocs/storage/whats-new-in-storage.md]: Yes, my bad, thank you!

**Review by eross-msft — APPROVED** [review]

**Pull request merged**